commisery/cli/__init__.py

```diff
@@ -122,7 +122,7 @@ def overview(ctx):
 @click.option("-d", "--description", help="Commit description")
 @click.option("-b", "--breaking-change", is_flag=True, help="This is a breaking change")
 @click.pass_context
-def commit(ctx, type, scope, description, breaking_change):
+def commit(ctx, type, scope, description, breaking_change):  # pylint: disable=W0622
```

Comment:
What is the warning that you're disabling?

Reply:
Argument count - will disable the warnings (as discussed)